ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```
@@ -564,6 +564,17 @@ tests:
       PACKET_PLAN: m3.large.x86
       TEST_SUITE: full
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-bond-periodic
+  cron: 45 03 */1 * 0-5
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        IS_BONDED=true
+        TEST_FUNC=test_networking
+      PACKET_PLAN: m3.large.x86
+      TEST_SUITE: full
+    workflow: assisted-baremetal
```

```suggestion
    workflow: assisted-baremetal
- as: e2e-metal-assisted-bond
  steps:
    cluster_profile: packet-assisted
    env:
      ASSISTED_CONFIG: |
        IS_BONDED=true
        TEST_FUNC=test_networking
      PACKET_PLAN: m3.large.x86
      TEST_SUITE: full
```
let's add a presubmit as well

---

OK

---

Isn't this presubmit ? https://github.com/openshift/release/blob/178f562bc56ce05717b3ce5de5bfba26d565921e/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml#L735

---

in `assisted-service` repo, not in `assisted-test-infra`